vocabularies/Hierarchy.xml

```diff
@@ -198,12 +198,21 @@ annotated with the `RecursiveHierarchy` annotation. The qualified name of the su
           <Annotation Term="Common.Experimental" />
           <Annotation Term="Core.Description" String="Number of [matching](#MatchCount) descendants a node has in the unlimited hierarchy" />
         </Property>
+        <Property Name="MessageCount" Type="Collection(Hierarchy.MessageCountType)" Nullable="false">
+          <Annotation Term="Common.Experimental" />
+          <Annotation Term="Core.Description" String="Number of messages for each severity a [matching](#MatchCount) descendant of a node has in the unlimited hierarchy" />
```

Comment:
Why do we refer to "matching" here? Are we only interested in messages for the matching nodes? If their ancestors are added as well, would we not show their messages?